flask_multipass_cern.py

```diff
@@ -16,9 +18,22 @@
 from flask_multipass.group import Group
 from flask_multipass.identity import IdentityProvider
 from flask_multipass.providers.authlib import AuthlibAuthProvider, _authlib_oauth
+from requests.adapters import HTTPAdapter
+from requests.exceptions import HTTPError
+from urllib3 import Retry
 
 
+CACHE_LONG_TTL = 172800
+CACHE_TTL = 1800
 CERN_OIDC_WELLKNOWN_URL = 'https://auth.cern.ch/auth/realms/cern/.well-known/openid-configuration'
+OIDC_RETRY_COUNT = 5
```

Comment:
`HTTP_RETRY_COUNT` or `API_RETRY_COUNT` since there's no OIDC in the API requests?